compiledb/parser.py

```diff
@@ -32,8 +32,8 @@
 compiler_wrappers = {"ccache", "icecc", "sccache"}
 
 # Leverage `make --print-directory` option
-make_enter_dir = re.compile(r"^\s*make\[\d+\]: Entering directory [`\'\"](?P<dir>.*)[`\'\"]\s*$")
-make_leave_dir = re.compile(r"^\s*make\[\d+\]: Leaving directory .*$")
+make_enter_dir = re.compile(r"^\s*make(?:\[\d+\])*: Entering directory [`\'\"](?P<dir>.*)[`\'\"]\s*$")
+make_leave_dir = re.compile(r"^\s*make(?:\[\d+\])*: Leaving directory .*$")
```

Comment:
I would suggest to allow at most one bracket pair:
```suggestion
make_enter_dir = re.compile(r"^\s*make(?:\[\d+\])?: Entering directory [`\'\"](?P<dir>.*)[`\'\"]\s*$")
make_leave_dir = re.compile(r"^\s*make(?:\[\d+\])?: Leaving directory .*$")
```